Jenkinsfile

```diff
@@ -228,16 +228,16 @@ pipeline {
             sh 'CUDA_VISIBLE_DEVICES="" python nemo_text_processing/inverse_text_normalization/inverse_normalize.py --lang=pt --text="dez " --cache_dir ${PT_TN_CACHE}'
           }
         }
-        stage('L0: ZH TN grammars') {
-         steps {
-            sh 'CUDA_VISIBLE_DEVICES="" python nemo_text_processing/text_normalization/normalize.py --lang=zh --text="你" --cache_dir ${ZH_TN_CACHE}'
-          }
-        }
-        stage('L0: ZH ITN grammars') {
-          steps {
-            sh 'CUDA_VISIBLE_DEVICES="" python nemo_text_processing/inverse_text_normalization/inverse_normalize.py --lang=zh --text="二零零二年一月二十八日 " --cache_dir ${ZH_TN_CACHE}'
-          }
-        }
+        // stage('L0: ZH TN grammars') {
+        //  steps {
+        //     sh 'CUDA_VISIBLE_DEVICES="" python nemo_text_processing/text_normalization/normalize.py --lang=zh --text="你" --cache_dir ${ZH_TN_CACHE}'
+        //   }
+        // }
+        // stage('L0: ZH ITN grammars') {
+        //   steps {
+        //     sh 'CUDA_VISIBLE_DEVICES="" python nemo_text_processing/inverse_text_normalization/inverse_normalize.py --lang=zh --text="二零零二年一月二十八日 " --cache_dir ${ZH_TN_CACHE}'
+        //   }
+        // }
```

Comment:
@alexcui-nvidia why commented out?

Reply:
I created zh tnitn separately. It was originally running for several languages all together. I got zh out, starting from line 270.

Reply:
@BuyuanCui please remove if not needed?